db/seeds.rb

```diff
@@ -5,3 +5,14 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
+require 'faker'
+
+Pokemon.destroy_all
+
+Pokemon.create!(
```

Comment:
Can you try this without the '!'. Like: `Pokemon.create`
also, since you installed Faker, you could use faker to seed the db multiple pokemons at once, example:
`Pokemon.create(
 name: Faker::Games::Pokemon.name,
  description: Faker::Games::Pokemon.move,
### You could generate a pokemon world location, but I think we want real locations.
  location: Faker::Address.full_address,
### You can decide the price range
  price: rand(100..200)
)`